install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -213,6 +213,11 @@ data:
   enforce-ingress-https: {{ .Values.ingressController.enforceHttps | quote }}
   enable-ingress-secrets-sync: {{ .Values.ingressController.secretsNamespace.sync | quote }}
   ingress-secrets-namespace: {{ .Values.ingressController.secretsNamespace.name | quote }}
+  {{- if .Values.ingressController.ingressLBAnnotations }}
+  ingress-lb-annotations: {{ .Values.ingressController.ingressLBAnnotations | join " " | quote }}
+  {{- else }}
+  ingress-lb-annotations: {{ .Values.ingressController.ingressLBAnnotationsDefault | join " " | quote }}
```

Comment:
same as above, we can remove this else block, as the default value is available in go code anyway.